app/src/app/components/login/UserAuthForm.tsx

```diff
@@ -32,7 +32,7 @@ export const UserAuthForm = ({ className, ...props }: UserAuthFormProps) => {
         windowNavigate(loginRoute);
       }
     }
-  }, [user?.token]);
+  }, []);
```

Comment:
This change is to fix redirect on login, which was not working. The BasicAuthForm was redirecting to the requested redirect location on login but this component was picking up the user token change and also redirecting - to "/". I'm not sure if this a race condition that has recently appeared? I *think* this change is ok because the idea is just that anyone browsing to "/login" should get redirected to root immediately if they're already logged in - so this code should only run once, on component load. 

For other auth methods - both github and montagu auth methods should mean we get redirected back to /redirect on login, so they shouldn't be affected by this change.